pkg/resources/user.go

```diff
@@ -226,126 +216,207 @@ func User() *schema.Resource {
 	}
 }
 
-func ImportUser(ctx context.Context, d *schema.ResourceData, meta any) ([]*schema.ResourceData, error) {
-	logging.DebugLogger.Printf("[DEBUG] Starting user import")
-	client := meta.(*provider.Context).Client
-	id, err := sdk.ParseAccountObjectIdentifier(d.Id())
-	if err != nil {
-		return nil, err
-	}
+func ServiceUser() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: GetCreateUserFunc(sdk.UserTypeService),
+		UpdateContext: GetUpdateUserFunc(sdk.UserTypeService),
+		ReadContext:   GetReadUserFunc(sdk.UserTypeService, true),
+		DeleteContext: DeleteUser,
+		Description:   "Resource used to manage service user objects. For more information, check [user documentation](https://docs.snowflake.com/en/sql-reference/commands-user-role).",
```

Comment:
It should be https://docs.snowflake.com/en/sql-reference/sql/create-user. The same in other user schemas.

Reply:
It was set to this for user resource because of this link: https://docs.snowflake.com/en/sql-reference/commands-user-role#user-management. Changed to https://docs.snowflake.com/en/sql-reference/commands-user-role#user-management because it contains the list of all the commands, not only create.